### DISCUSSION TITLE
does bromite have a bottom tab like in chrome ?

### DISCUSSION THREAD

**Discussion by Dhia-zorai** [bromite/bromite#2371] [Q&A]
couldn't find anything about it.
![IMG_20221002_150935](https://user-images.githubusercontent.com/114868460/193460350-72806f51-33d5-49ad-86d5-6959b9c6da21.jpg)

**Comment by ghost** [comment]


**Reply by n-buna404** [reply]
chrome://flags/#enable-tab-groups
set it to enabled
